### ISSUE TITLE
ErrorView state/view not refreshed unless view is popped in special case

### ISSUE THREAD

**Issue by drewvolz** [StoDevX/AAO-React-Native#220]
If you load the Financials and Courses views while the view ErrorView is present, then your view of either the Courses or Financials view is blocked after you log-in with the ErrorView on the stack. The state/view of the screen you are _not_ on does not get refreshed after logging in -- only the view that we log-in on is loaded. The user has to pop the navigator and return back to the SIS view if they would like to see and update that view. 

**In short:**
Home -> SIS -> tap both tabs -> hit 'open settings' -> login -> tap the other tab -> can't access -> go back home -> load SIS -> can access

This is unaffected by actually heading to Settings and logging in.

**Comment by drewvolz** [comment]
Not a shipping bug -- does not cause a crash, or loss of data. Simply an annoyance in an edge case. I say ship our current bug fixes without this one for now.

**Comment by hawkrives** [comment]
Oh drat. Yes, that's an unintentional side effect of how it's designed. 

I'll think about it on the way in.

**Comment by hawkrives** [comment]
Has this been avoided by pushing a view over the entire screen from the bottom? I think this is no longer an issue.

**Comment by hawkrives** [comment]
Nope, still an issue.

**Comment by hawkrives** [comment]
I'm taking this – I think it will be resolved by one of my redux branches, when I move login state into a global store, so both screens can access a canonical state.
